server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Server",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha ./test/**"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
There is probably a decision to take regarding licenses. I think the expo project has none at the moment

Reply:
Ok, il will remove it